index.html

```diff
@@ -222,34 +243,35 @@ <h2>Multi-Post Stories</h2>
     <!-- SEE PROJECT BUTTON -->
     <button class="btn">See project</button>
     </div>
-
+    </div>
+  </section>
     <section class="contact-form">
-   
-      <h2 class="contact-us">Contact Us</h2>
+      <h2 class="contact-us">Let's chat!</h2>
       <p class="paragraph3">If you have an application you are
         interested in developing, a feature that you
         need built or a project that needs coding.
         I'd love to help with it </p>
-    
       <form action="https://formspree.io/forms/meqdknwb/integration" method="post" class="form">
-      <input type="text" id="name" name="name" placeholder="First name" maxlength="30" required>
+      <input type="text" id="full-name" name="full-name" placeholder="Full name" maxlength="30" required>
+      <div class="desktop-contact-input-fields">
+      <input type="text" id="first-name" name="first-name" placeholder="First name" maxlength="30" required>
+      <input type="text" id="last-name" name="last-name" placeholder="Last name" maxlength="30" required>
+      </div>
       <input type="email" id="email" name="email" placeholder="Example@email.com" required>
       <textarea type="text" id="Message" name="message" placeholder="Write your message here" maxlength="500" required></textarea>
-      <button class="btn2">See project</button>
+      <button type="submit" class="btn2">Send</button>
     </form>
   </section>
   <!-- line4 -->
    <hr>
-    
   <footer class="section-footer">
       <ul class="social-icons">
         <li><a href=""><img src="./img/Vector.png" alt="github"></a></li>
         <li><a href=""><img src="./img/Frame.png" alt="Linkedin"></a></li>
         <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
         <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
-        <li><a href=""><img src="./imG/M.png" alt="medium"></a></li>
+        <li><a href=""><img src="./img/M.png" alt="medium"></a></li>
       </ul>
-    
       <p class="email-text">emailhere@email.com </p>
```

Comment:
- [ ] [Optional] I would suggest you please remove the white horizontal line as if you see the Figma design then you will not get that line.
### your design
![his](https://user-images.githubusercontent.com/81727988/200112145-74309f2c-5176-475f-a934-f9603974b0ce.png)

Reply:
ok thank you.